docs/ANONYMITY_NETWORKS.md

```diff
@@ -1,149 +1,176 @@
 ```
+# file /etc/tor/torrc
 HiddenServiceDir /var/lib/tor/data/monero
-HiddenServicePort 18081 127.0.0.1:18081
+# HiddenServicePort 18081 127.0.0.1:18081 # rpc port
+HiddenServicePort 18083 127.0.0.1:18083   # tx port
 ```
+and restart `sudo systemctl restart tor.service`. This will generate your `.onion` address in `/var/lib/tor/data/monero/hostname`. 
+
+I2P must be configured with a standard server tunnel. Configuration differs by I2P implementation.
+
+
+### Allow inbound RPC
 
-Then the wallet will be configured to use a Tor/i2p address:
+The clients `monero-wallet-cli` and `monero-wallet-rpc` can connect to a daemon over AN. The daemon must provide a hidden service on free port that redirects the request to your local rpc port `--rpc-bind-ip`.
 ```
---proxy 127.0.0.1:9050
---daemon-address rveahdfho7wo4b2m.onion
+HiddenServicePort 18081 127.0.0.1:18081 # rpc port
 ```
+Now not only localhost rpcs, but remote rpcs incoming from AN via port 18081 can use your service. Use with caution.
 
-The proxy must match the address type - a Tor proxy will not work properly with
-i2p addresses, etc.
+If you wish to run a public rpc service, further options for `monerod` are availble that restrict available commands, add rpc-ssl or require authentication. You can also require e.g. [Client Authoriztion](https://community.torproject.org/onion-services/advanced/) on the AN side.
 
-i2p and onion addresses provide the information necessary to authenticate and
-encrypt the connection from end-to-end. If desired, SSL can also be applied to
-the connection with `--daemon-address https://rveahdfho7wo4b2m.onion` which
-requires a server certificate that is signed by a "root" certificate on the
-machine running the wallet. Alternatively, `--daemon-cert-file` can be used to
-specify a certificate to authenticate the server.
 
-Proxies can also be used to connect to "clearnet" (ipv4 addresses or ICANN
-domains), but `--daemon-cert-file` _must_ be used for authentication and
-encryption.
+### Connect to remote daemon
 
-### Network Types
+You can use a client to do rpc calls against a remote `monerod` peer. The wallet will be configured to use a onion/i2p address:
+```
+--proxy 127.0.0.1:9050             # tor or i2p entry port
+--daemon-address rveahdfho.onion   # tor or
+--daemon-address cmeua5767.b32.i2p # i2p
+```
+The onion and i2p addresses provide the information necessary to authenticate and encrypt the connection end-to-end, which is inherent to the address design. A _remote_ daemon is untrusted by default, unless `--trusted-daemon` is set. A _local_ daemon is trusted unless option `--untrusted-daemon` is set.
 
-#### Tor & I2P
+If desired, SSL can also be applied to the connection with `--daemon-address https://rveahdfho.onion` which requires a server certificate that is signed by a "root" certificate on the machine running the wallet. Alternatively, `--daemon-cert-file` can be used to specify a certificate to authenticate the server.
```

Comment:
Mention that there is no benefit if `tord`/`i2pd` and `monerod` are running on the same machine. The AN provides encryption to the AN daemon; SSL is only useful if `monerod` is on a different box than the AN service.